core/src/main/scala/com/databricks/labs/remorph/parsers/intermediate/extensions.scala

```diff
@@ -236,3 +236,11 @@ case class ColumnAliases(input: Relation, aliases: Seq[Id]) extends RelationComm
 
 case class TableFunction(functionCall: Expression) extends Relation {}
 case class Lateral(expr: Relation) extends Relation {}
+
+case class CreateProcedure(
+    name: String,
+    parameters: Seq[FunctionParameter],
+    variables: Option[Seq[SetVariable]],
```

Comment:
It's merely a detail but `Option[Seq[X]]` is redundant in most cases (it appears here and there in the codebase but we'll have to clean that up eventually). There is rarely any meaningful difference between `None` and `Some(Seq())`. So we should prefer going for simply `Seq[X]` in general.